DCS_STATUS/StatusVmon904HV.py

```diff
@@ -0,0 +1,1266 @@
+                	SortListV.append(internalListV)
+                                                                                                                
+                #print(SortListV)
+                SortListV = sorted(SortListV, key=lambda elementV: elementV[0])
+                #print(SortListV)
+                
+                for refillV in range(len(vmonOnlyT)):
+                	vmonOnlyT[refillV]=SortListV[refillV][0]
+                	vmonOnlyV[refillV]=SortListV[refillV][1]
+                	vmonOnlyTDate[refillV]=SortListV[refillV][2]
+                                                                                                                
+                #print(vmonOnlyT)
+                #print(vmonOnlyV)
+               
+		if (len(vmonOnlyT))==0:
+			print("vmonOnlyT lenght", len(vmonOnlyT))
+                	print("------------------------------------------------------------------")
+                	print("ERROR: there are no HV voltage data for chamber "+ chamberList[indexB]+ " channel "+ channelList[indexC])
+                	print("------------------------------------------------------------------")
+                
+                	file = open("HVErr.log", "w") 
+                	file.write("ERROR: there are no HV voltage data for chamber "+ chamberList[indexB]+ " channel "+ channelList[indexC]) 
+                	file.close() 
+                
+			#counter = counter + 1 	
+			#I put an anomalous voltage value if there is no data
+                        vmonOnlyT.append(0.)
+                        vmonOnlyTDate.append(0.)
+                        vmonOnlyV.append(-1000000000)
+                		
+                	#continue
+
+ 
+                #rescale the negative times
+                negativeStartV = vmonOnlyT[0]
+                if vmonOnlyT[0] < 0:
+                	for iterTimeV in range(len(vmonOnlyT)):
+                		vmonOnlyT[iterTimeV] = vmonOnlyT[iterTimeV] + negativeStartV*(-1)
+                #print(vmonOnlyT)
+
+
+                #Vmontg1 = ROOT.TGraph(n,x,y); x and y is the name of arrays with numbers of time and V
+                Vmontg1 = ROOT.TGraph(len(vmonOnlyT),vmonOnlyTDate,vmonOnlyV)
+                Vmontg1.SetLineColor(2)
+                Vmontg1.SetLineWidth(4)
+                Vmontg1.SetMarkerColor(4)
+                Vmontg1.SetMarkerStyle(21)
+		Vmontg1.SetMarkerSize(1)
+                Vmontg1.SetName("HV_VmonChamber"+chamberList[indexB]+"_"+channelList[indexC]+"_UTC_time")
+                Vmontg1.SetTitle("HV_VmonChamber"+chamberList[indexB]+"_"+channelList[indexC]+"_UTC_time")
+                #Vmontg1.GetXaxis().SetTitle("time [s]")
+                Vmontg1.GetYaxis().SetTitle("Vmon "+chamberList[indexB]+" "+channelList[indexC]+" [V]")
+                #Vmontg1.Draw("ACP")
+		Vmontg1.GetXaxis().SetTimeDisplay(1)
+                Vmontg1.GetXaxis().SetTimeFormat("#splitline{%y-%m-%d}{%H:%M:%S}%F1970-01-01 00:00:00")
```

Comment:
clean up the whitespace here (convert all `tab` characters to an appropriate number of spaces,